02-peer-protocol.md

```diff
@@ -113,6 +122,10 @@ FIXME: Describe Dangerous feature bit for larger channel amounts.
 
 #### Requirements
 
+A sending node MUST ensure that the `chain-hash` value be a valid genesis block
```

Comment:
A sending node MUST ensure that the `chain-hash` value identifies the chain they they wish to open the channel within.  For the Bitcoin blockchain, the `chain-hash` value MUST be (encoded in hex):
000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f.